docs/tutorials/using-conponents.md

```diff
@@ -0,0 +1,106 @@
+---
+Level: Beginner
+---
+# Using components
+The best option you should consider when you plan to use an ADF component and want to learn the details of its usage, is always to check the documentation for the component you are looking to use. More in general, there are three different ways to use, extend and configure an ADF component:
+1. Configuration properties.
+2. Event listeners.
+3. Content projection / HTML extensions.
+
+In this tutorial you are going to see a practical example for each approach. As an example, the [Login component](https://alfresco.github.io/adf-component-catalog/components/LoginComponent.html) will be used.
+
+## Configuration properties
+Angular components can easily be configured via properties in the HTML template. In this example we will act on the "Remember me" check and "Need Help?" + "Register" links in the footer of the Login component.
+
+To prepare the task, be sure you have and ADF application up and running by executing `npm start` in a terminal, from the root folder of the project. Access to the login page using your browser and edit the `login.component.html` file stored into the `src/app/.../login` folder. The content of the `login.component.html` file should look like the following source code.
```

Comment:
"...you have *an* ADF application up and running..."

Reply:
Changed. Thank you.